lib/rules/selector-max-compound-selectors/__tests__/index.js

```diff
@@ -184,10 +200,16 @@ testRule({
 			message: messages.expected('.burgers .bacon a', 2),
 			line: 2,
 			column: 1,
+			endLine: 2,
+			endColumn: 18,
 		},
 		{
 			code: '.cd { .de { .fg {} } }',
 			message: messages.expected('.cd .de .fg', 2),
+			line: 1,
+			column: 13,
+			endLine: 1,
+			endColumn: 19,
```

Comment:
[note] The accuracy of the end column is lost due to `postcss-resolve-nested-selector`. It should be helpful that the end column would point to just a selector instead of the rule node, but this is not the case. See the example below.

Expected:

```scss
.cd { .de { .fg {} } }
/*          ^^^
            ↑  ↑
            13 16
*/
```

Actual:

```scss
.cd { .de { .fg {} } }
/*          ^^^^^^
            ↑     ↑
            13    19
*/
```